ipset.go

```diff
@@ -99,6 +112,23 @@ func (s *IPSet) RemoveSet(b *IPSet) {
 	}
 }
 
+// Invert updates s to contain the opposite of is current contents.
+func (s *IPSet) Invert() {
+	s.toInOnly()
+	s.out = s.in
+	s.in = []IPRange{
+		IPPrefix{IP: IPv4(0, 0, 0, 0), Bits: 0}.Range(),
+		IPPrefix{IP: IPv6Unspecified(), Bits: 0}.Range(),
```

Comment:
Should we iterate through the `s.in`, discover all IPv6 zones contained therein, and add those here? Should we have IPv6 at all if `s.in` only contains IPv4?

Either way, we should document what universe we're taking the complement in.

Reply:
Hmmm. IPSet doesn't really deal with zones at all right now (e.g. compaction will smush together IPs across zones), so I think I'm going to turn this into a bug to figure out what to do, and we can fix all of IPSet at once. Until then, leaving as-is.